tests/integ/scala/test_dataframe_suite.py

```diff
@@ -2586,21 +2614,25 @@ def test_replace(session):
 
     # replace NaN
     Utils.check_answer(
-        TestData.null_data3(session).na.replace({float("nan"): 11}, subset=["flo"]),
+        TestData.null_data3(session, local_testing_mode).na.replace(
+            {float("nan"): 11.0}, subset=["flo"]
```

Comment:
why we change from using `11` to `11.0`?

Reply:
Same reason as above.